src/zeit/content/cp/area.py

```diff
@@ -197,6 +197,8 @@ def elasticsearch_raw_query(self, value):
         # Ensure a consistent indentation for raw query
         if value is not None:
             value = json.dumps(json.loads(value), indent=2, ensure_ascii=False)
+            if 'query' not in value:
+                raise ValueError('Top-level key "query" is required.')
```

Comment:
Gute Idee! 👍

Aber wird denn `elasticsearch_raw_query` schon "in the wild" genutzt, wo wir das noch umschreiben oder ein fallback einbauen müssten?  Oder sind das nur ganz wenige, also wenn überhaupt?

Reply:
Jo mein Argument war, Sonja fängt _jetzt grade_ an die ersten Queries zu schreiben, insofern dürfte es eigentlich noch ziemlich genau gar keine in Production irgendwo geben.